apps/api/src/app/shared/framework/idempotency.interceptor.ts

```diff
@@ -183,7 +218,7 @@ export class IdempotencyInterceptor implements NestInterceptor {
 
     //already seen the request return cached response
     if (parsed.status === ReqStatusEnum.ERROR) {
-      Logger.error(`returning cached error response. key:${idempotencyKey}`, LOG_CONTEXT);
+      Logger.verbose(`returning cached error response. key: "${idempotencyKey}"`, LOG_CONTEXT);
```

Comment:
No need to log these as errors, as they are technically not a server error and are client errors. We should only log them in verbose mode.